Makefile.core.mk

```
@@ -85,7 +85,12 @@ site:
 snips:
 	@scripts/gen_snips.sh
 
-gen: tidy-go format-go update-gateway-version snips
+format-spelling:
+	@echo "Sorting the .spelling file..."
+	@sort .spelling -o .spelling
```

```suggestion
	@sort --ignore-case .spelling -o .spelling
```

Agree with Ian on this one. Otherwise, I don't think this is an issue. I'll defer to @frankbu and/or @ericvn for this but don't see an issue with having a sort (and this would be enforced via the gencheck test)

---

Revised, thanks for the suggestion!